packages/xo-server/src/xapi/mixins/patching.mjs

```diff
@@ -493,9 +493,7 @@ export default {
   },
 
   @decorateWith(deferrable)
-  async rollingPoolUpdate($defer, { xsCredentials } = {}) {
-    const isXcp = _isXcp(this.pool.$master)
-
+  async roolingPullReboot($defer, { xsCredentials, beforeEvacuateAny, beforeEvacuateHost } = {}){
```

Comment:
I think, `rollingPollReboot` shoud be on `mixin/pool.mjs` and not on `mixin/patching.mjs`